src/org/mozilla/javascript/NativeObject.java

```diff
@@ -507,6 +513,30 @@ public Object execIdCall(
                     Scriptable desc = obj.getOwnPropertyDescriptor(cx, nameArg);
                     return desc == null ? Undefined.instance : desc;
                 }
+            case ConstructorId_getOwnPropertyDescriptors:
+                {
+                    Object arg = args.length < 1 ? Undefined.instance : args[0];
+                    // TODO(norris): There's a deeper issue here if
```

Comment:
A super minor thing but I don't think that we should add a new TODO comment when copying code, especially one that I bet is 10+ years old and assigned to someone who hasn't worked on this project for a very long time ;-)

Reply:
ref. https://github.com/mozilla/rhino/pull/1193/commits/2b80a8ab7d6a89039c464367e87b057cc99c3109 I removed. I will fixup in a few days.